src/hint_processor/builtin_hint_processor/field_arithmetic.rs

```diff
@@ -88,8 +88,8 @@ pub fn u384_get_square_root(
     };
 
     if !&x.is_zero() && !(success_x ^ success_gx) {
-        return Err(HintError::AssertionFailed(String::from(
-            "assert success_x + success_gx ==1",
+        return Err(HintError::AssertionFailed(Box::new(
+            "assert success_x + success_gx ==1".to_string(),
```

Comment:
```suggestion
        return Err(HintError::AssertionFailed(
            "assert success_x + success_gx ==1".to_string().into_boxed_string(),
```

Reply:
Didn't know of this. Thanks for sharing!